packages/eslint-plugin/docs/rules/no-deprecated.mdx

```diff
@@ -14,7 +14,7 @@ It's best to avoid using code marked as deprecated.
 This rule reports on any references to code marked as `@deprecated`.
 
 :::note
-[TypeScript recognizes the `@deprecated` tag](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#deprecated) and visualizes deprecated code with a ~strikethrough~.
+[TypeScript recognizes the `@deprecated` tag](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#deprecated) and editors usually visualizes deprecated code with a ~strikethrough~.
```

Comment:
There's a minor typo
```suggestion
[TypeScript recognizes the `@deprecated` tag](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#deprecated) and editors usually visualize deprecated code with a ~strikethrough~.
```

But I'm wondering if we want to tweak it a bit more to emphasize that the editor uses TS's deprecation information in order to create the strikethrough...
```suggestion
[TypeScript recognizes the `@deprecated` tag](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#deprecated), allowing most editors to visualize deprecated code with a ~strikethrough~.
```
or
```suggestion
[TypeScript recognizes the `@deprecated` tag](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#deprecated), allowing editors to visually indicate deprecated code — usually with a ~strikethrough~.
```

***

What do you think?

Reply:
I agree it's a good idea --- I've applied the last suggestion.